assets/css/app.css

```diff
@@ -39,3 +39,48 @@
   100% { opacity: 0; }
 }
 
+/* PETAL css styles, see "How to install Petal Components" at https://petal.build/components */
```

Comment:
@SimonLab could you please add a section to https://github.com/dwyl/learn-tailwind below [`Tailwind` in `Phoenix`](https://github.com/dwyl/learn-tailwind#part-2-tailwind-in-phoenix)
summarising the steps to add [**/petal.build/components**](https://petal.build/components) to a **`Phoenix`** project? 🙏